modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-platform-prepare-end-to-end_{context}"]
+= End-to-end procedures for updating clusters in a disconnected environment
+
+If you have deployed spoke clusters with distributed unit (DU) profiles using the GitOps ZTP with Topology Aware Lifecycle Operator pipeline described in "Deploying distributed units at scale in a disconnected environment", this procedure tells you how to upgrade your spoke clusters and Operators.
+
+[id="talo-platform-prepare-for-update_{context}"]
+== Preparing for the updates
+
+If both the hub and the spoke clusters are running {product-title} 4.9, you need to update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
+
+[id="talo-platform-prepare-for-update-env-setup_{context}"]
+== Setting up the environment
+
+If you deployed ZTP in a disconnected environment as instructed in the "Preparing the disconnected environment" section in "Deploying distributed units at scale in a disconnected environment", you need to do the following steps before TALO can successfully perform update operations:
+
+TALO can perform both platform and Operator updates:
+
+* For platform updates, you have to do the following:
++
+. Mirror the desired {product-title} image repository. For more information about how to mirror an image repository, see the "Mirroring the OpenShift Container Platform image repository" section.
++
+[IMPORTANT]
+====
+The `imageContentSources` section, which can be retrieved with the `oc adm release mirror` command, needs to be saved for platform updates.
+For example, this is the output of the `imageContentSources` which contains the information of your mirrors:
+
+[source,terminal]
+----
+imageContentSources:
+ - mirrors:
+   - mirror-ocp-registry.ibmcloud.io.cpak:5000/openshift-release-dev/openshift4
+   source: quay.io/openshift-release-dev/ocp-release
+ - mirrors:
+   - mirror-ocp-registry.ibmcloud.io.cpak:5000/openshift-release-dev/openshift4
+   source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+====
+
+. Prepare the update graph. You have two options to prepare the update graph:
+
+.. Use the OpenShift Update Service.
++
+For more information about how to set up the graph on the hub cluster, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#build-the-graph-data-init-container[Build the graph data init container].
+
+.. Make a local copy of the upstream graph and host the update graph on a http or https server in the disconnected environment to which the spoke cluster has access. To download the update graph, use the following command:
```

Comment:
This first sentence is pretty confusing. It mentions an upstream graph and update graph, and the "to which" wording is ambiguous.
```suggestion
.. Make a local copy of the upstream graph. Host the update graph on an `http` or `https` server in the disconnected environment that has access to the spoke cluster. To download the update graph, use the following command:
```